package.json

```diff
@@ -51,11 +53,17 @@
   "jest": {
```

Comment:
Do we need to [configure parser](https://github.com/prettier/prettier#parser) for prettier?

Reply:
The default in Prettier works fine with Flow.